packages/manager/modules/iam/src/components/conditions/conditionType/date/dateCondition.controller.js

```diff
@@ -26,4 +31,11 @@ export default class IAMConditionDateController {
       this.updateConditionValues();
     }
   }
+
+  scrollCalendarIntoView() {
+    const calendarElt = this.$element.find('.flatpickr-calendar')[0];
+    if (calendarElt) {
+      calendarElt.scrollIntoView();
+    }
+  }
```

Comment:
As you repeat it in each component, it should be moved into a dedicated service

```suggestion
  scrollCalendarIntoView() {
    this.elementUiService.scrollIntoView('.flatpickr-calendar');
}
```
with that code
```js
export default class ElementUiService {

/* @ngInject */
constructor($element) {
    this.$element = $element;
}

scrollIntoView(elementName) {
   this.$element.find(elementName)[0]?.scrollIntoView();
}
}
myModule.service('elementUiService', ElementUiService)
```